.github/actions/buildkite/action.yml

```diff
@@ -0,0 +1,64 @@
+---
+name: Buildkite Pipeline GitHub Action
+description: A GitHub Action for triggering a build on a Buildkite pipeline.
+inputs:
+  vaultUrl:
+    description: 'Vault URL'
+    required: true
+  vaultRoleId:
+    description: 'Vault role ID'
+    required: true
+  vaultSecretId:
+    description: 'Vault secret ID'
+    required: true
+  secret:
+    description: 'Vault secret with the token field.'
+    default: secret/observability-team/ci/buildkite-automation
+    required: false
+  org:
+    description: 'BuildKite org to interact with.'
+    default: elastic
+    required: false
+  pipeline:
+    description: 'BuildKite pipeline to interact with.'
+    required: true
+  waitFor:
+    description: 'Whether to wait for the build to finish.'
+    default: false
+    required: false
+  printBuildLogs:
+    description: 'Whether to wait for the build to finish.'
+    default: false
+    required: false
+  buildEnvVars:
```

Comment:
This is a json based format, I'd like to explore using 

```
          buildEnvVars: |
            commit=abderg
            message=my-message
            org=my-org
            something=my super duper variable
```

rather than

```
buildEnvVars: { "commit": "abderg", "message": "my-message", "org": "my-org", "something": "my super duper variable"} 
```

Reply:
https://github.com/elastic/apm-pipeline-library/pull/2033/commits/3622e2d14cabfcfa77db91f4accca1b38ead9a2e

Reply:
See https://github.com/elastic/apm-pipeline-library/pull/2033/commits/d56115a5bc4bf7c9109008a6823a0973e55286b5